models/games-model.js

```diff
@@ -13,3 +13,19 @@ exports.selectCategories = () => {
             return result.rows;
         });
 };
+
+exports.selectReviews = () => {
+    return db
+        .query(
+            `
+                SELECT reviews.*, CAST(COUNT( comments.comment_id ) AS int) AS comment_count FROM reviews
+                INNER JOIN comments
```

Comment:
Would opt for an alternative JOIN here to include any articles that do not have any associated articles